Formula/curl.rb

```diff
@@ -67,6 +77,21 @@ def install
     args << (build.with?("gssapi") ? "--with-gssapi" : "--without-gssapi")
     args << (build.with?("rtmpdump") ? "--with-librtmp" : "--without-librtmp")
 
+    if build.with? "libidn2"
+      resource("libidn2").stage do
+        system "./configure", "--disable-dependency-tracking",
+                              "--disable-silent-rules",
+                              "--prefix=#{libexec}/vendor/libidn2",
```

Comment:
Generally for vendoring static is a bit better, if it works, since it only has to be installed in the buildpath temporarily never to be heard from again.